tuxemon/cli/parameter.py

```diff
@@ -0,0 +1,14 @@
+from __future__ import annotations
+
+import dataclasses
+
+
+@dataclasses.dataclass
+class Parameter:
+    """
+    Used to categorize and manage parameters for commands
+
+    """
+
+    name: str
+    klass: str
```

Comment:
Ahh, the classical German fallback XD. Is this actually used anywhere?

Reply:
no not at the moment.  it was being used for autocomplete, and will be used to provide suggestions.  i think it can be removed

Reply:
removed